ui/src/__mocks__/api.ts

```diff
@@ -0,0 +1,26 @@
+import { Org } from "../org"
```

Comment:
Do we have to set up this boilerplate every time we want to mock an endpoint call?

Reply:
We only have to make one mock of the `api` module, but to make sure we use it in a particular test context we have to include `jest.mock("./api")` in the file.